django_addanother/templates/django_addanother/related_widget_wrapper.html

```diff
@@ -2,9 +2,22 @@
 <div class="related-widget-wrapper">
     {{ widget }}
 
+    {% if edit_related_url %}
+    &nbsp;
+    <a class="related-widget-wrapper-link change-related" id="change_id_{{ name }}"
+        href=""
+        data-href-template="{{ edit_related_url }}?{{ url_params }}"
+        title="{% blocktrans %}Edit related{% endblocktrans %}">
+        <img src="{% static edit_icon %}" alt="{% trans 'Edit' %}"/>
+    </a>
+    {% endif %}
+
+    {% if add_related_url %}
+    &nbsp;
```

Comment:
It probably makes in render better by default, I can live with it even though I'd prefer this in a separate commit.